packages/react-composites/tests/app/chat/LiveTestApp.tsx

```diff
@@ -56,7 +56,8 @@ export const LiveTestApp = (): JSX.Element => {
 
   React.useEffect(() => {
     if (adapter && uploadedFiles.length) {
-      uploadedFiles.forEach((file) => {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      uploadedFiles.forEach((file: any) => {
```

Comment:
this should be type where it is defined, i.e.
```diff
-  const uploadedFiles = React.useMemo(() => (params.uploadedFiles ? JSON.parse(params.uploadedFiles) : []), []);
+  const uploadedFiles: any[] = React.useMemo(() => (params.uploadedFiles ? JSON.parse(params.uploadedFiles) : []), []);
```

Then this should have a strong type, should it be `ActiveFileUpload` ? (cc @vhuseinova-msft )
```ts
const uploadedFiles: ActiveFileUpload[] = React.useMemo(() => (params.uploadedFiles ? JSON.parse(params.uploadedFiles) : []), []);
```

Reply:
ActiveFileUpload seems to be missing a lot of props like: 
name, extension, url, error.
Is there another Type that is more appropriate? @vhuseinova-msft

Reply:
After checking this more with Edward, it looks like the `uploadedFiles` uses properties from `FileMetadata` and `ActiveFileUpload`. This code needs to be reviewed and updated to use correct types/values.

Reply:
Created a task item under Chat to refactor this outside of this PR.